lib/templates/android/inc/_attributes_getter_setter.liquid

```diff
@@ -0,0 +1,39 @@
+{%- for attribute in entity.attributes %}
+
+    {% if attribute.realm_read_only.size == 0 -%}
+        {%- comment %} ******* CONVERT TYPE CAPTURE ******* {% endcomment -%}
+        {%- capture convert_type -%}
+          {%- if attribute.realm_read_only.size == 0 and attribute.enum_type.size > 0 -%}
+            String
+          {%- else -%}
+            {%- if params.use_wrappers == true -%}
+              {%- include 'inc/wrapper_type_converter' -%}
+            {%- else -%}
+              {%- include 'inc/type_converter' -%}
+            {%- endif -%}
+          {%- endif -%}
+        {%- endcapture -%}
+        {%- capture primitives -%}
+          {%- include 'inc/primitives' -%}
+        {%- endcapture -%}
+        {%- capture annotation -%}{%- if params.support_annotations == true -%}
+        {%- if params.use_wrappers == true or attribute.enum_type.size > 0 or primitives == "false" -%}
+        @android.support.annotation.{% if attribute.optional == true %}Nullable{% else %}NonNull{% endif -%}
+          {%- if attribute.support_annotation.size > 0 %}-{% endif -%}
```

Comment:
Change the separator char to "|" as comma are used in Android annotations (ex : @android.support.annotation.IntRange(from=0,to=255)). See commit aff60995efb261570aabb7cba4faa9e4dcade490